### ISSUE TITLE
wp-rocket customizations

### ISSUE THREAD

**Issue by jlyon** [proudcity/wp-proudcity#2192]
Update the wp-rocket configuration site-wide:
- [ ] Ensure that Combine JavaScript files is checked (screenshot below)
- [ ] Caching issues @kevindherman add details

![Screen Shot 2023-01-31 at 12 48 14 PM](https://user-images.githubusercontent.com/550909/215879991-82ed0600-13af-48b8-8137-996586ba46ee.png)

**Comment by kevindherman** [comment]
@jlyon the caching issues it that we're getting a lot of tickets from customers that they're publishing changes and need to clear site cache in order to see them.

**Comment by lukefretwell** [comment]
@jlyon just a note that when I activate that for proudcity.com, the nav icon links don't work.

**Comment by curtismchale** [comment]
@lukefretwell yes we removed it because of JS issues last week around the nav links and maps for this issue: https://github.com/proudcity/wp-proudcity/issues/2185

In the call I believe I said it "fixes" the issue, but doesn't really fix the underlying JS issue we have been having. I need time to dig into the JS issue to find it and stop it from happening. It's possible @jlyon or @aschmoe have more insight if they really dig into the JS issue and can find the `var` that's not defined and fix it. I'll have to go digging and testing to find it.

**Comment by curtismchale** [comment]
@lukefretwell @jlyon I did some testing with the latest version of WP Rocket on beta and we have a bunch of error messages as outlined in this comment.

https://github.com/proudcity/wp-proudcity/issues/2196#issuecomment-1421148912

Combine **should not** currently be activated until we have these issues resolved.
